decidim/decidim
# Fix processes creation form with stats, metrics and announcements

#### :tophat: What? Why?

When creating a new process and not selecting the "show statistics" and "show metrics" checkboxes, it still was selected. While reviewing it, I noticed that we have a similar problem  with the "announcements" field, where it isn't saved on creation. 

This PR fixes that. 

#### :pushpin: Related Issues
*Link your PR to an issue*

- Fixes #8909 
- Related to #5477

#### Testing

1. Sign in as admin
2. Create a new process. Don't select "show statistics" or "show metrics". Put somemthing in "announcement" field. 
3. Save it 
4. Go to the edit form, see that they're both selected and there isn't any text in announcement field. 


:hearts: Thank you!
